src/frontend/Komponenter/Behandling/VedtakOgBeregning/Skolepenger/OpphørSkolepenger/OpphørSkolepenger.tsx

```diff
@@ -0,0 +1,186 @@
+import { ISkoleårsperiodeSkolepenger } from '../../../../../App/typer/vedtak';
+import React, { Dispatch, SetStateAction, useMemo } from 'react';
+import styled from 'styled-components';
+import { useBehandling } from '../../../../../App/context/BehandlingContext';
+import { ListState } from '../../../../../App/hooks/felles/useListState';
+import { useApp } from '../../../../../App/context/AppContext';
+import { VEDTAK_OG_BEREGNING } from '../../Felles/konstanter';
+import navFarger from 'nav-frontend-core';
+import FjernKnappMedTekst from '../../../../../Felles/Knapper/FjernKnappMedTekst';
+import OpphørUtgiftsperiodeSkolepenger from './OpphørUtgiftsperiodeSkolepenger';
+import { beregnSkoleår, GyldigSkoleår } from '../skoleår';
+import KansellerKnapp from '../../../../../Felles/Knapper/KansellerKnapp';
+import { locateIndexToRestorePreviousItemInCurrentItems, oppdaterValideringsfeil } from '../utils';
+import SkoleårDelårsperiode from '../InnvilgetSkolepenger/SkoleårDelårsperiode';
+import { FormErrors } from '../../../../../App/hooks/felles/useFormState';
+import { InnvilgeVedtakForm } from '../InnvilgetSkolepenger/VedtaksformSkolepenger';
+
+const Skoleårsperiode = styled.div`
+    margin: 1rem;
+    margin-right: 0.5rem;
+    margin-left: 0rem;
+    padding: 1rem;
+    background-color: ${navFarger.navGraBakgrunn};
```

Comment:
```suggestion
    background-color: var(--navds-semantic-color-canvas-background);
```

Her kan man bruke `--navds-semantic-color-canvas-background` fra det [nye designsystemet](https://aksel.nav.no/designsystem/side/farge-tokens/semantiske). Kan brukes direkte siden `@navikt/ds-css` er importert i index.tsx.